skbio/sequence/_rna.py

```diff
@@ -136,6 +136,72 @@ def _motifs(self):
         return _motifs
 
     @stable(as_of="0.4.0")
+    def reverse_transcribe(self):
+        """Reverse transcribe RNA into DNA.
+
+        Uracil (U) is replaced with thymine (T) in the reverse transcribed
+        sequence.
+
+        Returns
+        -------
+        DNA
+            Reverse transcribed sequence.
```

Comment:
Can you add `(coding strand)` to the end of this description? We make note of this in `DNA.transcribe`.